digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -756,16 +763,29 @@ object DataFixture {
 
       OracleDatabase.withDynTransaction{
         //Obtain all existing RoadLinkId by AssetType and roadLinks
-        val assetCreated: Seq[Long] = dataImporter.getAllLinkIdByAsset(LanesNumberAssetTypeId, roadLinks.map(_.linkId))
+        val assetCreated = dataImporter.getAllLinkIdByAsset(LanesNumberAssetTypeId, roadLinks.map(_.linkId))
 
       println ("Total created previously      -> " + assetCreated.size)
 
       //Filter roadLink by Class
       val roadLinksFilteredByClass = roadLinks.filter(p => (p.administrativeClass == State))
       println ("Total RoadLink by State Class -> " + roadLinksFilteredByClass.size)
 
+      //Obtain asset with a road link type asset.Motorway or asset.Freeway with amount of lanes < 2
+      val roadLinkMotorway  = roadLinksFilteredByClass.filter(road => road.linkType == asset.Motorway  || road.linkType == asset.Freeway)
```

Comment:
Maybe it's better rename this "roadLinkMotorway", since we will obtain RoadLinks of two types, Motorway and Freeway.